controllers/cronset_controller.go

```diff
@@ -19,17 +19,18 @@ package controllers
 import (
 	"context"
 	"fmt"
+	"github.com/go-logr/logr"
```

Comment:
oh.. seems like a lot of people to use this library

Reply:
You originally used the logger that was in controller-runtime lib. the lib used this logr.